webui/react/src/App.tsx

```diff
@@ -97,7 +89,7 @@ const AppView: React.FC = () => {
         placement: 'bottomRight',
       });
     }
-  }, [ info ]);
+  }, [ analytics, info ]);
```

Comment:
seems unnecessary.
`  92:6  warning  React Hook useEffect has an unnecessary dependency: 'analytics'. Either exclude it or remove the dependency array. Outer scope values like 'analytics' aren't valid dependencies because mutating them doesn't re-render the component  react-hooks/exhaustive-deps
`

Reply:
oops that's outdated, great catch